### PR TITLE
Added FlashMessenger to authorization actions

### PR DESCRIPTION
**Pull request by BHenkemans**
# Description
Added FlashMessenger to authorization actions.

## Related issues/external references

Resolves GH-1825

## Types of changes

- [ ] Bug fix _(non-breaking change which fixes an issue)_
- [X] New feature _(non-breaking change which adds functionality)_
- [ ] Breaking change _(fix or feature that would cause existing functionality to change)_
- [ ] Documentation improvement _(no changes to code)_
- [ ] Other _(please specify)_

### CODE REVIEW

**Comment by BHenkemans** [comment]
A merge was the only way of solving the po-file problems. Rebasing never got me the right result :(

File: module/Decision/src/Controller/DecisionController.php [hunk 1]
```diff
@@ -155,10 +160,16 @@ public function searchAction(): ViewModel
         );
     }
 
-    public function authorizationsAction(): ViewModel
+    public function authorizationsAction(): Response|ViewModel
     {
         if (!$this->aclService->isAllowed('create', 'authorization')) {
-            throw new NotAllowedException($this->translator->translate('You are not allowed to authorize someone'));
+            $this->flashMessenger()->addErrorMessage($this->translator
+                ->translate('You are not allowed to authorize someone'));
```
Comment (tomudding) [inline module/Decision/src/Controller/DecisionController.php:167]: Please remove this such that we can assert that the `throw` is the only thing that will happen here.

File: module/Decision/src/Controller/DecisionController.php [hunk 2]
```diff
@@ -205,6 +213,10 @@ public function authorizationsAction(): ViewModel
     public function revokeAuthorizationAction(): Response|ViewModel
     {
         if (!$this->aclService->isAllowed('revoke', 'authorization')) {
+            $this->flashMessenger()->addErrorMessage($this->translator
+                ->translate('You are not allowed to revoke authorizations.'));
```
Comment (tomudding) [inline module/Decision/src/Controller/DecisionController.php:217]: Same here

**Review by tomudding — CHANGES_REQUESTED** [review]
> A merge was the only way of solving the po-file problems. Rebasing never got me the right result :(

It can happen; I usually select `theirs` and redo the translations on `ours` (assuming the PO editor you are using is capable enough to keep some history).

**Review by tomudding — APPROVED** [review]

**Comment by tomudding** [comment]
<3

**Pull request merged**